js/components/closeButton.jsx

```diff
@@ -6,12 +6,16 @@ import CONSTANTS from '../constants/constants';
 
 class CloseButton extends React.Component { // eslint-disable-line
   render() {
+    let { cssClass } = this.props;
     const {
       className,
-      cssClass,
+      controller,
       role,
       closeAction,
     } = this.props;
+    if (controller.state.readDirection === 'rtl') {
+      cssClass += ' oo-close-button-left';
```

Comment:
I suggest using `[dir=rtl]` in CSS instead of class names. This would be more readable and declarative